### PR TITLE
Image in English for IT document

### PR DESCRIPTION
**Pull request by yuyuankang**

### CODE REVIEW

File: docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md [hunk 1]
```diff
@@ -220,44 +220,6 @@ public class IoTDBAlignedSeriesQueryIT {
 }
 ```
 
-### Commands for starting IT
```
Comment (qiaojialin) [inline docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md:223]: why remove this?

Comment (irvine0109) [reply docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md:223]: The commands part is already described at this link https://github.com/apache/iotdb/blob/master/integration-test/README.md

Comment (irvine0109) [reply docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md:223]: Maybe we can put a link here. 
What do you think?

Comment (qiaojialin) [reply docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md:223]: +1 for a link

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**